web/test/integration/project/project.feature

```diff
@@ -16,3 +16,29 @@ Scenario: Ensure the create private project
     Given there is a document "test-private-project" with the json "test/data/private-project.json" in collection "projects"
     And the "/" page is open
     And the text "Demo private project" is not in the element ".project__list__title"
+
+Scenario: Ensure the public project with minium requirements
+    Given there is a document "guest-test-public-project" with the json "test/data/guest-public-project.json" in collection "projects"
+    And the "/" page is open
+    And the text "Demo guest public project" is in the element ".project__list__title"
+    And the text "2" is in the element ".mat-column-repository"
+    And the "/projects/guest-test-public-project" page is open
+    And the text "You do not have any monitors. Please create monitor" is in the element ".helper"
+    And the text "You do not have any repositories. Please connect repository" is in the element ".helper"
+
+Scenario: Ensure the public project with no data saved
+    Given there is a document "public-project-no-data" with the json "test/data/public-project-no-data.json" in collection "projects"
+    And the "/" page is open
+    And the text "Demo public project with no data" is in the element ".project__list__title"
+    And the "/projects/public-project-no-data" page is open
+    Then the text "No releases are available" is in the element ".info"
```

Comment:
This would read better as...

```gherkin
Given there is a document "public-project-no-data" with the json "test/data/public-project-no-data.json" in collection "projects"
When the "/" page is open
Then the text "Demo public project with no data" is in the element ".project__list__title"
When the "/projects/public-project-no-data" page is open
Then the text "No releases are available" is in the element ".info"
....
```

But some of the others might need to be updated also

Reply:
sure !! done